setup.py

```diff
@@ -60,6 +60,9 @@
               'idna>=2.0.0',
               'certifi',
           ],
+          'secure:python_version < "3.3"': [
```

Comment:
A few things here:

1. The python version here does not match the one in setup.cfg
2. As you might note, in setup.cfg `secure` has a couple of `python_version` checks but we don't replicate them here (specifically because of concerns for older versions of setuptools (which is exactly what you ran into).
  Maybe we should not be catering to a slightly larger number of setuptools versions, but I'm not sure being usable on those older versions is a bad thing either.